CHANGELOG.md

```diff
@@ -174,6 +174,139 @@
+- Pods are deleted across node upgrade (#32323)
+- Secure master -> node communication (#11816)
+- upgrading master doesn't upgrade kubectl (#32538)
+- Specific error message on failed rolling update issued by older kubectl against 1.4 master (#32751)
+- bump master cidr range from /30 to /29 (#32886)
+- non-hostNetwork daemonsets will almost always have a pod that fails to schedule (#32900)
+- Service loadBalancerSourceRanges doesn't respect updates (#33033)
+- disallow user to update loadbalancerSourceRanges (#33346)
+
+## Notable Changes to Existing Behavior
+
+### Deployments
+
+- ReplicaSets of paused Deployments are now scaled while the Deployment is paused. This is retroactive to existing Deployments.
+- When scaling a Deployment during a rollout, the ReplicaSets of all Deployments are now scaled proportionally based on the number of replicas they each have instead of only scaling the newest ReplicaSet.
+
+### kubectl rolling-update: < v1.4.0 client vs >=v1.4.0 cluster
+
+Old version kubectl's rolling-update command is compatible with Kubernetes 1.4 and higher only if you specify a new replication controller name. You will need to update to kubectl 1.4 or higher to use the rolling update command against a 1.4 cluster if you want to keep the original name, or you'll have to do two rolling updates.
+
+If you do happen to use old version kubectl's rolling update against a 1.4 cluster, it will fail, usually with an error message that will direct you here. If you saw that error, then don't worry, the operation succeeded except for the part where the new replication controller is renamed back to the old name. You can just do another rolling update using kubectl 1.4 or higher to change the name back: look for a replication controller that has the original name plus a random suffix.
+
+Unfortunately, there is a much rarer second possible failure mode: the replication controller gets renamed to the old name, but there is a duplicated set of pods in the cluster. kubectl will not report an error since it thinks its job is done.
+
+If this happens to you, you can wait at most 10 minutes for the replication controller to start a resync, the extra pods will then be deleted. Or, you can manually trigger a resync by change the replicas in the spec of the replication controller.
+
+### kubectl delete: < v1.4.0 client vs >=v1.4.0 cluster
+
+If you use an old version kubectl to delete a replication controller or replicaset, then after the delete command has returned, the replication controller or the replicaset will continue to exist in the key-value store for a short period of time (<1s). You probably will not notice any difference if you use kubectl manually, but you might notice it if you are using kubectl in a script.
+
+## Action Required Before Upgrading
+
+- If you are using Kubernetes to manage `docker` containers, please be aware Kubernetes has been validated to work with docker 1.9.1, docker 1.11.2 (#23397), and docker 1.12.0 (#28698)
```

Comment:
I am ok to include docker engine info here. But please include v1.10 since openshift might still use that version here. 

Then we can include each docker version's known issue separately in known issue above. Thanks!